client/my-sites/checkout/checkout/domain-details-form.jsx

```diff
@@ -154,6 +158,18 @@ export class DomainDetailsForm extends PureComponent {
 		return this.state.currentStep !== last( this.state.steps );
 	}
 
+	validateSteps() {
+		const updatedSteps = [ 'mainForm', ...this.getRequiredExtraSteps() ];
```

Comment:
Would be nice to use some `const` for this, as we have a few occurrences of it now. :)

Reply:
Thanks for testing @aidvu ! 

Agree on this point. `getRequiredExtraSteps()` could even return the array of steps (`[ 'mainForm', ... ]` etc), since that's all we're interested in. I'll make a note of it for the next refactor!